css/style.css

```diff
@@ -17,3 +17,119 @@ body {
  */
 
 
+* {
+    font-weight: lighter;
+}
+
+/* Navigation styles */
+.topnav img {
+    margin-left: 4rem;
+}
+
+.topnav {
+    display:flex;
+    justify-content: space-between;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0.5rem;
+    margin-right: 4rem;
+}
+
+li {
+    margin-left: 1.5rem;
+}
+
+li:first-of-type {
+    font-weight:700;
+}
+
+.nav-links {
+    display: flex;
+    justify-content: space-between;
+} 
+
+a:link, a:visited {
+    text-decoration: none;
+    color: rgb(119, 117, 117);
+}
+
+a:hover, a:focus {
+    color:rgb(230, 104, 8);
+}
+
+
+/* Hero styles */
+
+.hero {
+    /* Sizing */
+    width: 100vw;
+    height: 100vh;
+
+    /* Flexbox stuff */
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    /* Text styles */
+    text-align: center;
+    color:white;
+
+    /*Hero background image */
+    background-image: url("../img/first-background.jpg");
+    background-size: cover;
+}
+
+ /* Flexbox for icons */
+.icon-container {
+    display:flex;
+    justify-content: space-around;
+
+}
+
+.icons {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
```

Comment:
```suggestion
```
These child elements are already stacked vertically so we don't need to add flex column here. We can also centre the text by adding `text-align: center;` to the `.icon-container` rule instead